### PR TITLE
UI: Added workflow duration fields

### PR DESCRIPTION
**Pull request by dzannotti**
Added duration column per session in /workflows/:id
Added duration column per attempt in /sessions/:id
Added Last attempt duration field in brief (top view) in /sessions/:id

### CODE REVIEW

File: digdag-ui/console.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import moment from 'moment';
 import pako from 'pako';
 import path from 'path';
 import yaml from 'js-yaml';
+import Duration from 'duration';
```
Comment (dzannotti) [inline digdag-ui/console.jsx]: not too fond of this dep, but rather this than reinvent the wheel

**Comment by danielnorberg** [comment]
👍 LGTM!

**Comment by danielnorberg** [comment]
should we merge and deploy?

**Comment by dzannotti** [comment]
Yep, let's hold off on the release as i'm about to add another PR

**Pull request merged**